cellranger/3.1/Dockerfile

```diff
@@ -1,16 +1,17 @@
 FROM ubuntu:18.04
 MAINTAINER Alex Sickler (sicklera@email.chop.edu)
 
+ENV CELLRANGER_MAJ_VER 3.1
 ENV CELLRANGER_VERSION 3.1.0
```

Comment:
So what's going on here? We request the MINOR version (https://semver.org/) and we get a the most up to date patch? So if we run this docker today we're getting 3.1.0 but in a week the same curl command will give us 3.1.1?

Reply:
Sort of. I was testing this out and if the requested version isn't the most recent version, the grep won't be able to find the link since it's looking for the requested version. It would be easy to change to ignore the minor version and just download whatever minor version it gets, but then we would lose the ability to track the minor version.

Reply:
Yeah, it's kinda hard to tell what they do - if they end up with a `3.1.1`, whether they'll keep the `3.1.0` around as well, or we lose it and are forced to use a newer minor version

Reply:
Looking on their site and trying it, it looks like 5.0.0 isn't around anymore now that 5.0.1 is out. They seem to keep the minor versions around; 3.0.whatever and 3.1.0.